README.md

```diff
@@ -30,15 +30,15 @@ Maven repo
   <dependency>
     <groupId>io.appform.rules</groupId>
     <artifactId>json-rules</artifactId>
-    <version>1.0.12</version>
+    <version>LATEST</version>
```

Comment:
the artifact name needs to be changed now .. no?

Reply:
Done.